docs/governance/decred-constitution.md

```diff
@@ -1,48 +1,41 @@
 - To be a stakeholder, one must purchase one or more tickets, which entails locking a specific amount of coins. The amount of DCR that must be locked, also known as the ticket price, fluctuates as the system targets 40,960 tickets in the live pool.
-- After waiting for the ticket to mature, the ticket is entered into a lottery that runs once per block where five winning tickets gain the ability to vote on the previous block.
+- In each block, 5 tickets are called to vote, determined by a lottery [system](https://docs.decred.org/proof-of-stake/proof-of-stake/). When a ticket is called, a nominated wallet must actively respond with a vote. For a block to be accepted by the network it must include votes from at least 3 of the 5 tickets called. The blockchain cannot be extended without the active participation of stakeholders.
 - Stakeholders must wait an average of 28 days (8,192 blocks) to vote their tickets, and during this time the coins used to purchase the ticket remain locked. The wait may be much longer or shorter than the average of 28 days because the ticket selection process is pseudorandom. Tickets expire after approximately 142 days (40,960 blocks).
 - Stakeholder votes recorded in the blockchain are rewarded with 6% of each block subsidy, and each block can have up to 5 votes for a total of 30% of each block subsidy.
 - PoW receives 60% of each block subsidy, subject to the constraint that their subsidy scales linearly with the number of PoS votes included, e.g. including 3 of 5 votes reduces PoW subsidy to 60% of the maximum.
-- The votes themselves decide by majority decision whether the regular transaction tree of the previous block, including the PoW subsidy, is valid. Thus, if PoS voters vote against a particular PoW block, it destroys the PoW reward (and Treasury reward) and invalidates any regular transactions within that block.
-- Additional vote bits may be set when stakeholders submit votes, allowing stakeholders to vote on matters besides the previous block. The method of adopting changes to the consensus rules uses these additional vote bits.
+- The votes themselves decide by majority decision whether the regular transaction tree of the previous block, including the PoW subsidy, is valid. Thus, if PoS voters vote against a particular PoW block, it invalidates the PoW reward (and Treasury reward) and invalidates any regular transactions within that block.
+- Decred's process for [amending the consensus rules](https://docs.decred.org/governance/consensus-rule-voting/consensus-rules-voting/) is also driven by stakeholder voting. The process begins when at least 95% of PoW miners and 75% of PoS voters have upgraded their software to a new version with latent changes to the rules. Once these criteria are met, a voting period of 8,064 blocks (~4 weeks) begins, to decide whether the latent rule changes should be activated. 
```

Comment:
Consider replacing "Once these criteria are met" with "Once these conditions are met."

Also, I am a little cautious on the references to external documents. I would make sure you have an Inconsistency provision or similar provision such that in the event any external docs become inconsistent with the constitution, the constitution takes precedence.

Reply:
Also, include the phrasing "as of __/__/__" in case of any changes to external websites.

Reply:
The links to docs site should be safe as that's where the constitution also sits, critical details will not change in those pages without good reason (probably an approved proposal).